GoogleForCreators/web-stories-wp
# Dashboard: Applied design updates to Cooking template

## Summary

This PR applies the new design updates to the Cooking template.

## User-facing changes

Layout and text sizes have been update on the Cooking template based off the recommendations found in this [figma design doc](https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61).

## TODO
- We don't yet have the SVG grids imported into our shapes panel, so I used lines to manually lay out the grid.  In a follow up PR (when the SVG grids are imported), I'll update the template to use the SVG grids.

## Testing Instructions

1.) Go to "Explore Templates".
2.) Hover over the Cooking template and click on "View".
3.) Take a look at each template page in the detail view for Cooking.  Make sure images are showing up and the layout appears as shown in the figma updates.

---

Fixes #2472  

## Screenshots

<img src="https://user-images.githubusercontent.com/40646372/85472791-35f92200-b567-11ea-841e-ef9219400b7f.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85472828-43aea780-b567-11ea-8e60-153e392fcc45.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85472855-4f01d300-b567-11ea-9d22-01697051bd20.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85472907-6214a300-b567-11ea-8588-1749f79e6e47.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85472932-6b057480-b567-11ea-9f4d-0500a8712aea.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85472969-748edc80-b567-11ea-82dd-8eaeb89fa392.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85473000-7f497180-b567-11ea-8381-08aea581106c.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85476914-203b2b00-b56e-11ea-8e9e-cb1e75139644.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85473076-94be9b80-b567-11ea-88c6-bbcecb9d13d7.png" height="200">
